src/xcode/ENA/ENA/Source/Services/CoronaTestService/SubmissionTestType.swift

```diff
@@ -0,0 +1,36 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+enum SubmissionTestType: Equatable {
+	case registeredTest(CoronaTestType?)
+	case srs(SRSSubmissionType)
+}
+
+enum SRSSubmissionType: Equatable {
```

Comment:
Would like to add a minor comment, why we wrap the `SAP_Internal_SubmissionPayload.SubmissionType` with that `SRSSubmissionType` enum type. (Cause of equatability)